gce/sysprep/instance_setup.ps1

```diff
@@ -286,9 +243,12 @@ function Enable-RemoteDesktop {
       services.
```

Comment:
Aha, not sure why I missed that. Changed the rest to remove extra line as it makes long functions very long